pom.xml

```diff
@@ -48,11 +49,25 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-commons</artifactId>
+            <version>${junit.platform.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-engine</artifactId>
+            <version>${junit.platform.version}</version>
             <scope>test</scope>
         </dependency>
```

Comment:
The dependencies were added to pom.xml but not added to build.gradle. Therefore, `gradlew clean test` fails:
```
> Task :compileTestJava FAILED
C:\Projects\IDEAProjects\interview\src\test\java\com\devexperts\service\AccountServiceTransferTest.java:6: error: cannot find symbol
import org.junit.jupiter.api.Timeout;
                            ^
  symbol:   class Timeout
  location: package org.junit.jupiter.api
```

I think we should we move on with either maven or gradle.

Reply:
added proper dependencies for gradle. 
I'm not ready to move on just yet, however I agree it adds a lot of headache.
Side note: it's better to run `gradlew cleanTest test` so that gradle actually runs tests